File: scripts/extract-images.mjs
import fs from 'fs';

const html = fs.readFileSync('e:/Dreams Mantra/dreamz-about.html', 'utf8');
const re = /https:\/\/lh3\.googleusercontent\.com\/sitesv\/[^"'\s\\<>]+/g;
const urls = [...new Set(html.match(re) || [])];
urls.forEach((u) => console.log(u));
console.log('TOTAL', urls.length);

File: scripts/extract-images.py
import re
from pathlib import Path

html = Path(r"e:\Dreams Mantra\dreamz-about.html").read_text(encoding="utf-8", errors="ignore")
urls = set(re.findall(r"https://lh3\.googleusercontent\.com/sitesv/[^\"'\s\\<>]+", html))
for u in sorted(urls, key=len):
    print(u)
print("TOTAL", len(urls))
